### DISCUSSION TITLE
Storing UI settings?

### DISCUSSION THREAD

**Discussion by KoosWelling** [leozide/leocad#737] [Ideas]
Hello,
I was wondering, if it's possible to change the defaults for the UI in a settings file (or on exit)?
We would love to have:

- Snap XY on 1 stud
- Snap Z 1/20th stud
- Snap 90 degrees
- Projection: Orthographic

Would also be great, if standard spacing could be different, then 8mm?
Which I guess, is a constant in the program.

I also experience (sometimes) a lot of 'crashes'. And then it's a bit annoying to reset the UI settings again and again.

Regards,
Koos.

**Comment by leozide** [comment]
If you're using Windows, the app should save a file with some debugging information when it crashes, can you send me that file or some steps to reproduce the crashes?

**Reply by KoosWelling** [reply]
Tnxs Leozide, well, this is not the only software having an issue once in a while ;)
And yes I'm using Windows 10.

If I have a new crash, I'll send you the dump file.
A lot of times, when I get a crash information box, but if I ignore this (just don't press 'OK'), I could still continue (and ofc directly save my work). It's more rare, that the program directly exits.

But I was more interested about the possibility of changing standard settings?
Is that possible?
